### PR TITLE
build(react): Fix typescript errors in test files

### PR DESCRIPTION
**Pull request by sumitappt**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by sumitappt** [comment]
Screenshot from 2024-03-05 20-47-45
Screenshot from 2024-03-05 20-25-13

File: datahub-web-react/src/app/lineage/types.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ export type NodeData = {
     canEditLineage?: boolean;
     upstreamRelationships?: Array<LineageRelationship>;
     downstreamRelationships?: Array<LineageRelationship>;
-    health?: Health[];
+    health?: Maybe<Health[]>;
```
Comment (asikowitz) [inline datahub-web-react/src/app/lineage/types.ts:84]: Why was this change necessary? Can we replace usage with `health={health ?? undefined}` or something like that instead?

**Review by asikowitz — COMMENTED** [review]
Overall looks very reasonable to me. I'm gonna look into the type casts a little + the entity profile test failure, and I have one question

File: datahub-web-react/src/app/lineage/types.ts (hunk 1 above)
Comment (asikowitz) [reply datahub-web-react/src/app/lineage/types.ts:84]: Nvm, seems fine

**Review by asikowitz — APPROVED** [review]

**Comment by asikowitz** [comment]
Merging before smoke tests finish as this should only be frontend test / types changes
